prodmgr/parser.py

```diff
@@ -99,5 +99,20 @@ def create_parser():
         help='The hostname of the container image registry',
         default=DEFAULT_CONTAINER_REGISTRY_HOSTNAME,
     )
+    parser.add_argument(
+        '--deletion-container-name',
```

Comment:
I might be being a bit pedantic, but it should technically be the image version, not the container version.